### PR TITLE
Can use router functions in webflux

### PR DESCRIPTION
**Pull request by santitigaga**
## Description

Can use router functions in webflux and RestController the default option is router functions
## Category
- [x] Feature
- [ ] Fix

## Checklist
- [x] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [x] Automated tests are written
- [x] The documentation is up-to-date
- [ ] the version of the build.gradle, readme.md and gradle.properties was increased
- [x] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog

### CODE REVIEW

File: src/main/resources/entry-point/rest-webflux/router-functions/router.java.mustache [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@Configuration
+public class Router {
+
+@Bean
+public RouterFunction<ServerResponse> routerFunction(Handler handler){
+    return route(GET("/api/usecase/path"), handler::listenGETUseCase)
+    .andRoute(POST("/api/usecase/otherpath"),handler::listenPOSTUseCase)
+    .and(route(GET("/api/otherusercase/path"),handler::listenGETOtherUseCase));
+
```
Comment (juancgalvis) [inline src/main/resources/entry-point/rest-webflux/router-functions/router.java.mustache:18]: Remove line and indent

File: src/main/resources/entry-point/rest-webflux/router-functions/handler.java.mustache [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@RequiredArgsConstructor
+{{/lombok}}
+public class Handler {
+//private  final UseCase useCase;
+//private  final UseCase2 useCase2;
+{{^lombok}}
+public Handler() {
+}
+{{/lombok}}
+public Mono<ServerResponse> listenGETUseCase(ServerRequest serverRequest) {
```
Comment (juancgalvis) [inline src/main/resources/entry-point/rest-webflux/router-functions/handler.java.mustache]: Indent all returns in this file

File: src/main/resources/entry-point/rest-webflux/router-functions/router.java.mustache (hunk 1 above)
Comment (santitigaga) [reply src/main/resources/entry-point/rest-webflux/router-functions/router.java.mustache:18]: it's ok

File: src/main/resources/entry-point/rest-webflux/router-functions/handler.java.mustache (hunk 2 above)
Comment (santitigaga) [reply src/main/resources/entry-point/rest-webflux/router-functions/handler.java.mustache]: it's ok

**Review by juancgalvis — APPROVED** [review]
It's OK! Thanks

**Pull request merged**